pandapower/convert_format.py

```diff
@@ -275,10 +275,10 @@ def _convert_to_mw(net):
 
     for element, std_types in net.std_types.items():
         for std_type, parameters in std_types.items():
-            for parameter, value in parameters.items():

```

Comment:
In Python3.8, iteration over a dict while reducing/increasing it's size leads to `RuntimeError: dictionary changed size during iteration`